Formula/lgeneral.rb

```diff
@@ -16,10 +16,12 @@ class Lgeneral < Formula
     sha256 x86_64_linux:   "eefab1384276b2406cbdb286bfc730b3d434112f2d5ea8be6c3edb451d34f2e9"
   end
 
+  # setting deprecated, because brew audit fails build because or deprecated sdl_mixer
+  deprecate! date: "2023-02-24", because: :does_not_build
+
   depends_on "gettext"
   depends_on "sdl12-compat"
-  depends_on "sdl2"
-  depends_on "sdl2_mixer"
+  depends_on "sdl_mixer"
```

Comment:
We don't want to switch to old dependencies

Reply:
- **sdl_mixer** requires **sdl12-compat** which requires **sdl2**
- **sdl2_mixer** was set incorrect, never used

Reply:
So how do we get it to use `sdl2_mixer`?

Reply:
sdl2_mixer is wrong dependency here(added by my by mistake). The code does no use it.